src/main/java/nextstep/fp/Lambda.java

```diff
@@ -26,31 +26,21 @@ public void run() {
         }).start();
     }
 
+    public static int sumAll(List<Integer> numbers, Conditional conditional) {
+        return numbers.stream()
+                .filter(conditional::test)
+                .reduce(0, (a, b) -> a + b);
```

Comment:
사소하지만 다음과 같이 변경해도 좋을 것 같아요 :)
```suggestion
                .reduce(0, Integer::sum);
```